uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/LeaveRequestRights.php

```diff
@@ -88,9 +76,10 @@ public function canChangeDatesFor($contactID, $statusID, $requestType) {
    * @return bool
    */
   public function canChangeAbsenceTypeFor($contactID, $statusID) {
-    $leaveRequestStatuses = self::getLeaveRequestStatuses();
-    return $this->currentUserIsLeaveContact($contactID) &&
-           in_array($statusID, [$leaveRequestStatuses['awaiting_approval'], $leaveRequestStatuses['more_information_required']]);
+    $isOpenLeaveRequest = in_array($statusID, LeaveRequest::getOpenStatuses());
+    $isLeaveContactOrAdmin = $this->currentUserIsLeaveContact($contactID)|| $this->currentUserIsAdmin();
```

Comment:
Seems a space is missing between `$this->currentUserIsLeaveContact($contactID)` and `||`